README.md

```diff
@@ -31,6 +31,97 @@ GetText/DecorateFunctionMessage:
   Enabled: true
 ```
 
+## Cops
+### GetText/DecorateFunctionMessage
+This cop looks for any raise / fail functions and checks the user visible message is using gettext decoration with the _() function.
+This cop will make sure the message is decorated as well as checking that the formatting of the message is compliant according to the follow rules.
```

Comment:
Generally throughout this README, I would change most of these future tense uses ( "will make sure") to present tense("makes sure"); a comma is also needed after "decorated". I think that might be an American vs. UK style thing; we love commas here. :-)